TOPdeskPS/functions/Supporting Files/Get-TdOperatorGroup.ps1

```diff
@@ -0,0 +1,33 @@
+﻿function Get-TdOperatorGroup {
+    <#
+.SYNOPSIS
+    returns list of operator groups
+.DESCRIPTION
+    returns list of operator groups
+.EXAMPLE
+    PS C:\> Get-TdOperatorGroup
+    returns list of operator groups
+#>
+    [CmdletBinding(HelpUri = 'https://andrewpla.github.io/TOPdeskPS/commands/Get-TdOperatorGroup')]
+
+    param
+    (
+
```

Comment:
add name parameter

Reply:
this file is a duplicate see General API, added name anyway

Reply:
Go through the general api and remove the commands that don't belong there. These should be in Supporting Files

Reply:
fixed about a dozen duplicate/misplace files